doc/source/multi_run_plans.rst

```diff
@@ -0,0 +1,511 @@
+Multi-Run Plans
+===============
+
+This section is a brief tutorial on multi-run plans (introduced in Bluesky v1.6.0).
+A traditional single-run plan contains a set of instructions for running only one scan,
+which is assigned scan ID and UID. When multi-run plan is executed by Run Engine, multiple
+scans can be performed as part of a single plan. Data from each scan can be independently
+displayed and saved to the database via Databroker. The prior versions of Bluesky supported
+sequential execution of multiple scans within the plan. Starting from Bluesky v1.6.0, plans
+may include nested scans, i.e. the scans that can be interrupted, another scan executed and then the
+interrupted scan resumed and completed. Interruption can be initiated by the scan itself
+(simply by calling another scan from within the currently running scan) or externally (e.g. by triggering
+a suspender). The tutorial include brief explanation of the new Bluesky features that facilitate
+support of multi-run plans and examples that demonstrate implementation of the plans with
+sequential, nested and recursive calling of scans.
+
+Definition of a 'Scan'
+----------------------
+
+We define a scan as a block of code which generates a sequence of messages that control
+the data acquisition experiment. The scan starts with `open_run` and ends with `close_run` messages.
+Each scan is assigned distinct Scan ID and UID and produces a separate
+set of documents, including 'start' and 'stop' documents that are emitted and the processed by
+callbacks (such as BestEffortCallback) and saved to the database via Databroker (if enabled).
+
+In the plan, the scan may be defined by explicitely enclosing the code in `bps.open_run()` and
+`bps.close_run()` stubs:
+
+.. code-block:: python
+
+    # Using 'bps.open_run()' and 'bps.close_run()' stabs to define a scan
+
+    import bluesky.plan_stubs as bps
+    from bluesky import RunEngine
+
+    RE = RunEngine({})
+
+    def sample_plan():
+        ...
+        bps.open_run(md={})  # 'md' - metadata to be added to the 'start' document
```

Comment:
missing `yield from` here and below

Reply:
Thanks. Sorry I missed the obvious.

Reply:
No problem. An easy mistake to make---hopefully https://github.com/bluesky/bluesky/issues/1239 will help make it less easy to make.